sysvinit/freebsd/dsidentify

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# PROVIDE: dsidentify
+# REQUIRE: mountcritlocal
+# BEFORE:  cloudinitlocal
+
+. /etc/rc.subr
+
+PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+name="dsidentify"
+command="/usr/local/lib/cloud-init/ds-identify"
```

Comment:
i notice a lot of fixed paths. It makes me wonder if we can somehow replace them with `@LOCALBASE@` or similar, which the ports system would translate for us.